packages/frameworks-nuxt/src/runtime/lib/server.ts

```diff
@@ -0,0 +1,88 @@
+import { Auth, setEnvDefaults } from "@auth/core"
+import type { AuthConfig } from "@auth/core"
+import { getRequestFromEvent, getBasePath } from "../utils"
+import { defineEventHandler } from "h3"
+import type { H3Event } from "h3"
+import type { Session } from "@auth/core/types"
+
+/**
+ * Handle auth calls in Nuxt project.
+ *
+ *  @example
+ * ```ts title="server/routes/auth/[...].ts"
+ * import { NuxtAuthHandler } from "#auth"
+ * import GitHub from "@auth/nuxt/providers/github"
+ *
+ * export default defineEventHandler(async (event) => {
+ *    return NuxtAuthHandler({ providers: [GitHub] })
+ * })
+ * ```
+ *
+ * @param config AuthConfig
+ * @returns EventHandler
+ */
+export function NuxtAuthHandler(config: AuthConfig) {
+  return defineEventHandler(async (event) => {
+    /**
+     * If the request is a prerender request, do nothing.
+     */
+    if (
+      event.node.req.headers?.["x-nitro-prerender"] &&
+      import.meta.env.NODE_ENV === "prerender"
+    ) {
+      return
+    }
+
+    // Do not handle source maps
+    if (event.node.req.url?.includes(".js.map")) return
+
+    // Set missing items/ override items based on the environment variables
+    setEnvDefaults(process.env, config)
+
+    const request = await getRequestFromEvent(event)
+    config.trustHost ??= true
+    config.basePath ??= getBasePath(request)
```

Comment:
why don't we just set it as: 
```suggestion
    config.basePath ??= '/auth'
```
like other frameworks.

Reply:
OK